dao/src/main/resources/db/changelog/logs/ch-insert-into-users-Korzh.xml

```diff
@@ -5,7 +5,7 @@
     <changeSet id="Korzh-5" author="Nikita Korzh">
         <insert tableName="users">
             <column name = "date_of_registration" value="1970-01-01 00:00:00.000000"/>
-            <column name="email" value="green.city.ubs@gmail.coml"/>
```

Comment:
Why you changed existing change logs? That will be a cause of the Liquid Base checksum exception.

Reply:
it will be merged only when the database is dropped